api/scpca_portal/models/computed_file.py

```diff
@@ -354,6 +349,23 @@ def metadata_file_name(self):
     def zip_file_path(self):
         return settings.OUTPUT_DATA_PATH / self.s3_key
 
+    def get_multiplexed_computed_files(self):
+        """
+        Return computed file objects for all associated multiplexed samples.
+        The self computed file object is used as a template in building identical computed files,
+        differentiated only by sample, so that they can be bulk created at the same time.
+        """
+        multiplexed_samples = list(self.sample.multiplexed_with_samples) + [self.sample]
+        self.sample = None
```

Comment:
The reason why I went in this direction is because `model_to_dict` doesn't gracefully handle Foreign Keys. What it does is pass (in this case) the sample_id through, which causes an error when creating a new object because the Django ORM is expecting an object and not an id (integer). Combining the samples to iterate over all of them and dropping the sample from the self object allows for the self object to function as a template from which all associated computed_files can be built.